ui-tests/cypress/support/command/command.js

```diff
@@ -28,4 +28,26 @@ Cypress.Commands.add('uploadRequest', ({ method, url, body, headers, auth }) =>
   });
 });
 
+Cypress.Commands.add('uploadRequest', ({ method, url, body, headers, auth }) => {
```

Comment:
isn't it the same as line 8?

Reply:
sorry. I must have slipped this out when I rebased the branch